### DISCUSSION TITLE
PURGE CDN Cache

### DISCUSSION THREAD

**Discussion by Georg7** [firebase/firebase-js-sdk#4960] [Q&A]
I [stumbled upon ](https://stackoverflow.com/questions/52787090/firebase-hosting-with-cloud-functions-how-to-purge-refresh-cdn-cache) a suggestion that one can ```PURGE``` the Firebase CDN cache

```curl -X PURGE https://yoursite.com/path```

I get a status 200 response from that but the cache remains unchanged. There doesn't seem to be any documentation so I'm looking for guidance if/how the CDN cache can be cleared for a specific url path.

**Comment by hsubox76** [comment]
Just to clarify, that example is referring to a Storage url, are you also trying to clear the cache for a Storage url?

**Reply by Georg7** [reply]
My use case is dynamic content being loaded on `https://yoursite.com/path` which is cached with 
```res.set('Cache-Control', 'public, max-age=300, s-maxage=600')```. 

The user may then update the content that is being loaded on `/path`. In this case I want to purge `/path` so the updated content can be loaded.

**Reply by Georg7** [reply]
@hsubox76 - do you have any feedback/input on the above? Thanks!
